### PR TITLE
Removed Ruby Plugin references

### PR DESCRIPTION
**Pull request by KBbitsP**
Hi @sbrossie @reshmabidikar

### CODE REVIEW

File: userguide/tutorials/aws-multitier.adoc [hunk 1]
```diff
@@ -431,6 +431,7 @@ The `diagnostic` option of the `kpm` command creates an extensive report for a g
   --killbill-api-credentials=KEY SECRET \
   --kaui-web-path=/var/lib/tomcat/webapps2 \
   --killbill-url=http://127.0.0.1:8080 \
+  --account-export=ACCOUNT_ID \
   --database-host=DBURL:3306
 ```
```
Comment (reshmabidikar) [inline userguide/tutorials/aws-multitier.adoc:435]: Could you revert this change? As [discussed](https://github.com/killbill/killbill-docs/pull/464#issuecomment-1418455253), @jdmooney would be making this change as part of his work.

File: userguide/tutorials/aws-singletier.adoc [hunk 2]
```diff
@@ -227,6 +227,7 @@ The `diagnostic` option of the `kpm` command creates an extensive report for a g
   --killbill-api-credentials=KEY SECRET \
   --kaui-web-path=/var/lib/tomcat/webapps2 \
   --killbill-url=http://127.0.0.1:8080 \
+  --account-export=ACCOUNT_ID \
   --database-host=127.0.0.1:3306
```
Comment (reshmabidikar) [inline userguide/tutorials/aws-singletier.adoc:230]: Same comment as above, Jim would be incorporating this change as part of his work.

File: userguide/tutorials/database_migrations.adoc [hunk 3]
```diff
@@ -70,7 +69,7 @@ For example, to download all migration files for Kill Bill between 0.16.12 and 0
 kpm migrations killbill killbill-0.16.12 killbill-0.18.0
```
Comment (reshmabidikar) [inline userguide/tutorials/database_migrations.adoc:69]: Does anything need to be done to get rid of the above alert?

**Review by reshmabidikar — COMMENTED** [review]
Looks good. I noticed that the https://docs.killbill.io/latest/database_migrations.html#_ruby section is not deleted. Could you take a look at this section and delete it if it is not required?

File: userguide/tutorials/aws-multitier.adoc (hunk 1 above)
Comment (KBbitsP) [reply userguide/tutorials/aws-multitier.adoc:435]: Hi @reshmabidikar removed  the account export line.

File: userguide/tutorials/aws-singletier.adoc (hunk 2 above)
Comment (KBbitsP) [reply userguide/tutorials/aws-singletier.adoc:230]: Hi @reshmabidikar this one too.

File: userguide/tutorials/database_migrations.adoc (hunk 3 above)
Comment (KBbitsP) [reply userguide/tutorials/database_migrations.adoc:69]: Hi @reshmabidikar I don't think so. I just removed couple of words in the text, rest all formatting is the same.

**Review by reshmabidikar — APPROVED** [review]

File: userguide/tutorials/database_migrations.adoc (hunk 3 above)
Comment (reshmabidikar) [reply userguide/tutorials/database_migrations.adoc:69]: Looking into the [details ](https://github.com/killbill/killbill-docs/security/code-scanning/5248) of this alert looks like the code needs to be formatted using `[source, bash]`. I'll merge the PR for now, but maybe this can be addressed in a subsequent PR?
